src/main/java/app/studnicki/ox/ConsoleUserInterface.java

```diff
@@ -82,6 +81,10 @@ public int fieldId(int limit) {
     }
   }
 
+  /**
+   * Prints a message as an error
```

Comment:
Its also up to you but as far as I know doc descriptions usually ends with "." ;)

Reply:
Added the magical dot ;) Also that cleared my checkstyle report.